### PR TITLE
Apply saved filters

### PR DESCRIPTION
**Pull request by sharph**
Fix filter selection drop down to allow filter loading multiple times

### CODE REVIEW

File: web/app/scripts/saved-filters/saved-filters-controller.js [hunk 1]
```diff
@@ -27,8 +27,8 @@
         }
 
         // Activates the selected filter
-        function viewFilter(/*savedFilter -- commented out for jshint*/) {
-            // TODO: implement view filter logic
+        function viewFilter(savedFilter) {
+            $scope.$emit('driver.savedFilters:filterSelected', savedFilter.filter_json);
```
Comment (kshepard) [inline web/app/scripts/saved-filters/saved-filters-controller.js]: I'm guessing jshint won't like that underscore.

**Comment by kshepard** [comment]
This looks good. One more thing it needs to do though is when `view` is clicked from the dashboard widget, it should switch over to the map view.

**Comment by kshepard** [comment]
+1

**Comment by kshepard** [comment]
I just tried this out on staging, and there appears to be a problem with applying filters from the dashboard. If I clear fliters, go to the dashboard and apply a filter, it brings me to the map page, but with no filters applied.

**Comment by sharph** [comment]
I'll have a look.

**Pull request merged**